CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Grafana Mimir
 
 * [ENHANCEMENT] Compactor: Add `reason` label to `cortex_compactor_runs_failed_total`. The value can be `shutdown` or `error`. #4012
+* [ENHANCEMENT] OTLP: Add support for converting OTel Exponential Histograms to Prometheus Native Histograms.
```

Comment:
@krajorama What's the impact for Mimir (`main` version) once we'll start receiving native histograms via OTLP?

Reply:
Nothing. Mimir currently does not accept native histograms, in fact it doesn't know about that kind of data type at all in its implementation of the remote write protocol. Even after https://github.com/grafana/mimir/pull/4041 is merged, it would just ignore any histograms.